.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Pull Request Description
```

Comment:
```suggestion
```
We may not need this header for the PR template. IMO, it is already verbose with the information below.